.github/workflows/main.yml

```diff
@@ -76,17 +80,14 @@ jobs:
           key: ${{ runner.os }}-vscode-${{ hashFiles('**/vscode-version.ts') }}
           restore-keys: |
             ${{ runner.os }}-vscode-
-      - name: Build Extension
-        uses: gradle/gradle-build-action@v2
-        with:
-          arguments: extension:build
-        env:
-          NODE_OPTIONS: "--max-old-space-size=4096"
```

Comment:
We don't need to run `./gradlew extension:build` anymore. We run `./gradlew build -x test` in line 39, which already executes `extension:build`, so we just need to upload and reuse the artifacts 🙂